docs/manual/treefile.md

```diff
@@ -61,6 +61,15 @@ It supports the following parameters:
  * `packages-$basearch`: Array of strings, optional: Set of installed packages, used
     only if $basearch matches the target architecture name.
 
+ * `blacklist-packages`: Array of strings, optional: Each entry in this list is a package name
```

Comment:
Yeah, I think I agree with @lucab re. naming. Since this is essentially equivalent to `exclude=`, maybe `exclude-packages` instead to make the connection obvious. WDYT?